### DISCUSSION TITLE
Update session when I refresh the page

### DISCUSSION THREAD

**Discussion by rahmat-dev** [nextauthjs/next-auth#7705] [Help] [answered]
Hi, I use next-auth CredentialsProvider with JWT strategy. When I update user data from database and then I refresh the page, user data in session not updated.
Also when I delete user from database, session still exists when I refresh the page.

How to update or delete the session when I refresh the page?

**Comment by JasperAlexander** [comment, marked as answer]
Is the answer not in the docs? [https://next-auth.js.org/getting-started/client#updating-the-session](https://next-auth.js.org/getting-started/client#updating-the-session)
